askama_derive/src/generator.rs

```diff
@@ -234,6 +237,29 @@ impl<'a> Generator<'a> {
         self.writeln("}");
         self.writeln("}");
     }
+    
+    // Implement Actix-web's `Responder`.
+    fn impl_actix_web_responder(&mut self) {
+        //let lifetime = syn::Lifetime::new("'askama", Span::call_site());
```

Comment:
Bah, I forgot to remove these. They can be taken out,`actix_web::Responder` doesn't need 'em.